04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+\subsection{Future works Concepts}
+
+The disaster is a random function, meaning that it is possible to predict it with the current game.\\
+
+There are multiple future works ideas that would be developed in the design aspect such as; a deterministic disaster which would be  designed as a straight line that accumulates during the days, once the threshold has been met then the disaster would occur. Another deterministic disaster idea would be to link the disaster’s magnitude to time. Thus, agents will have to learn from past disaster occurrences, creating a memory. Agents will start learning from past disasters and will be able to forecast the next one. Therefore, the agents will be able to forecast that if a disaster did not surface for a long period of time, then the magnitude of the next disaster will be much larger than the previous one.\\
+
+Another Idea is that agents can invest into forecasting. Therefore, agents will have access to a history of past disasters, which would be utilised by the agents to learn and gain knowledge in order to predict the upcoming disaster. The larger the amount of resources invested; the agents will have access to a larger history of previous disasters. Therefore, agents will acquire knowledge, when investing resources.\\
+
+\section{Common Pool}
+\subsection{Common Pool Background}
+
+We are designing a game where 6-islands (agents) are having access to a common-pool resource. In micro-level, each island intends to maximise its utility while in macro-level, all the islands want to maintain sustainability and be protected by the upcoming disaster. The above description specifies a collective action problem where n-agents (6 in our case) are seeking access to a common pool resource that is sufficient to satisfy some agents but not to satisfy all of them.\\
+
+We have implemented a Linear Public Goods (LPG) Game where all the agents individually own some resources and try to maintain and increase them, by foraging, but also contribute a part of them to the common pool. This common pool is used for the payment of expenses of the institutional roles and for the protection towards the disaster.\\
+
+In more detail, the common pool is represented by a structure with following characteristics:
+
+\begin{itemize}
+    \item Common Pool Threshold
+    \item Amount of Resources 
+\end{itemize}
+
+According to the specifications of an LPG Game, the agents should be able to perform the following actions:  
+
+\begin{itemize}
+    \item Request Resource from the Common Pool 
+    \item Contribute Resource to the Common Pool 
+\end{itemize}
+
+And the Common Pool is responsible for two more actions: 
+
+\begin{itemize}
+    \item Mitigate Disaster
+    \item Deplete islands after a disaster
+\end{itemize}
+
+\newpage
+\subsection{functions within the Common pool}
```

Comment:
can we please NOT inclue the functions in this PR?

also, why don't you start with capital letter for the titles???